apps/api/src/domain/games/memory-match/queries/memory-match-queries.fetch-by-id.e2e.spec.ts

```diff
@@ -50,24 +59,66 @@ const publishedRound = buildTestInstance(MemoryMatchRound, {
     name: buildMultilingualTextWithSingleItem('published round'),
 });
 
+const unpublishedRound = buildTestInstance(MemoryMatchRound, {
+    id: buildDummyUuid(123),
+    isPublished: false,
+});
+
 /**
  * TODO[https://coscrad.atlassian.net/browse/CWEBJIRA-305]
  * Return unpublished rounds to admin users.
  */
 describe(`when querying for a memory match round: fetch by Id`, () => {
-    const testDatabaseName = generateDatabaseNameForTestSuite();
-
     let app: INestApplication;
 
     let databaseProvider: ArangoDatabaseProvider;
 
     let memoryMatchRepository: IMemoryMatchRepository;
 
+    const setItUp = async (testUserWithGroups: CoscradUserWithGroups) => {
+        const mockConfigService = buildMockConfigService(
+            {
+                // TODO can we fix this?
+                ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+                GLOBAL_PREFIX: '',
```

Comment:
Does this really lead to the `null` in the rendered media item URLs?

Reply:
You need to inject both the `GLOBAL_API_PREFIX` and the `NODE_PORT` in the mock config.
<img width="1363" height="639" alt="image" src="https://github.com/user-attachments/assets/8bbd9379-ecf5-4704-b058-08f333c25a75" />